src/diffpy/utils/diffraction_objects.py

```diff
@@ -169,32 +175,53 @@ def __eq__(self, other):
         return True
 
     def __add__(self, other):
-        summed = deepcopy(self)
-        if isinstance(other, int) or isinstance(other, float) or isinstance(other, np.ndarray):
-            summed.on_tth[1] = self.on_tth[1] + other
-            summed.on_q[1] = self.on_q[1] + other
-        elif not isinstance(other, DiffractionObject):
-            raise TypeError("I only know how to sum two DiffractionObject objects")
-        elif self.on_tth[0].all() != other.on_tth[0].all():
-            raise RuntimeError(x_grid_emsg)
-        else:
-            summed.on_tth[1] = self.on_tth[1] + other.on_tth[1]
-            summed.on_q[1] = self.on_q[1] + other.on_q[1]
-        return summed
+        """Add a scalar value or another DiffractionObject to the xarrays of
+        the DiffractionObject.
 
-    def __radd__(self, other):
```

Comment:
radd i think we don't need?

Reply:
you are correct - added `__radd__` back using ` __radd__ = __add__` 


please see a new test below for `do + scalar` as well as `scalar + do`